spec/integration/report_spec.rb

```diff
@@ -4,7 +4,7 @@
   before :each do
     webauth = double('WebAuth', :login => 'sunetid', :attributes => {'DISPLAYNAME' => 'Example User'}, :privgroup => ADMIN_GROUPS.first)
     @current_user = User.find_or_create_by_webauth(webauth)
-    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@current_user)
+    allow_any_instance_of(CatalogController).to receive(:current_user).and_return(@current_user)
```

Comment:
Strange strange behavior happening here. Where `ApplicationController` will not receive the stubbed method, yet when specifying a descendant of `ApplicationController`, `ReportController` or `CatalogController` that method is received. Not sure why this is and looking into it for a bit turned up no results. Any ideas?